panther_analysis_tool/main.py

```diff
@@ -1160,6 +1181,41 @@ def contains_invalid_field_set(analysis_spec: Any) -> List[str]:
     return invalid_fields
 
 
+def contains_invalid_table_names(analysis_spec: Any, analysis_id: str) -> List[str]:
```

Comment:
Can we move this to another file? I acknowledge that most validation stuff lives in main.py right now. But I am on a personal mission to not add more stuff to main if it doesn't have to be there

Reply:
Done, and moved one other validation method out as well